app/screens/home/search/search.tsx

```diff
@@ -235,6 +245,17 @@ const SearchScreen = ({teamId}: Props) => {
             />
         );
     }
+    const autocomplete = useMemo(() => (
```

Comment:
In theory, memoizing this shouldn't give any extra performance, since it is just the component, and react should take care of that. Nevertheless, I haven't done any proper research on that.

Reply:
From our previous discussions, that is what I recall.  I believe I had added some logic in there at first and it was removed leaving only the component.